.changeset/large-mirrors-teach.md

```diff
@@ -0,0 +1,13 @@
+---
+"@comet/cms-site": major
+---
+
+Fix an issue where the admin preview could break a block's styling and HTML structure
+
+This was caused by a `div` added around every block to enable the selection and highlighting of the block in the admin preview.
+The `div` is still present but now uses `display: contents`, so its effect should be minimal.
```

Comment:
```suggestion
Fix an issue where the block preview could break a block's styling and HTML structure

This was caused by a `div` added around every block to enable the selection and highlighting of the block in the block preview.
The `div` is still present but now uses `display: contents`, so its effect should be minimal.
```

We use the terms block preview and site preview, so we probably shouldn't introduce a new term admin preview